ludwig/utils/fs_utils.py

```diff
@@ -192,9 +192,6 @@ def copy(src, tgt, recursive=False):
 def makedirs(url, exist_ok=False):
     fs, path = get_fs_and_path(url)
     fs.makedirs(path, exist_ok=exist_ok)
-    if not path_exists(url):
-        with fsspec.open(url, mode="wb"):
-            pass
```

Comment:
I thought this was here to ensure that in case we can't make a "dir", we just create a file in write mode since S3 storage works that way? But I guess this may break for other types of storage. I just checked and it seems like fsspec does seem to automatically do this exact thing for filesystems that need it, so I guess it should be ok to remove this